### DISCUSSION TITLE
メッシュ・サブメッシュ結合ツールを使わないで変換できますか(VRChat→vrm)

### DISCUSSION THREAD

**Discussion by otoneco** [esperecyan/VRMConverterForVRChat#49] [Q&A]
オンオフの切り替えとかは可能でしょうか。

**Comment by esperecyan** [comment]
現在そのようなオプションはありません。

VRMへの変換は、何もわからない人向けの、あくまでも簡易的な変換の提供を目的としています。
細かい調整が必要あれば、UniVRMを直接利用したエクスポートをオススメしますが、いかがでしょうか。

**Reply by otoneco** [reply]
ありがとうございます。
色々試してみます。
